build: error Can't make a request in offline mode ("https://registry.yarnpkg.com/analytics-node/-/analytics-node-3.0.0.tgz")

On fresh gceworker from today, running `make build` gave me this. Running it again then worked.

```
build/node-run.sh -C pkg/ui yarn install --offline
go install -v ./pkg/cmd/prereqs
Submodule 'c-deps/cryptopp' (https://github.com/cockroachdb/cryptopp.git) registered for path 'c-deps/cryptopp'
Submodule 'c-deps/googletest' (https://github.com/cockroachdb/googletest) registered for path 'c-deps/googletest'
Submodule 'c-deps/jemalloc' (https://github.com/cockroachdb/jemalloc.git) registered for path 'c-deps/jemalloc'
Submodule 'c-deps/protobuf' (https://github.com/cockroachdb/protobuf.git) registered for path 'c-deps/protobuf'
Submodule 'c-deps/rocksdb' (https://github.com/cockroachdb/rocksdb.git) registered for path 'c-deps/rocksdb'
Submodule 'c-deps/snappy' (https://github.com/cockroachdb/snappy.git) registered for path 'c-deps/snappy'
Submodule 'pkg/ui/yarn-vendor' (https://github.com/cockroachdb/yarn-vendored) registered for path 'pkg/ui/yarn-vendor'
Submodule 'vendor' (https://github.com/cockroachdb/vendored) registered for path 'vendor'
Cloning into '/home/tschottdorf/go/src/github.com/cockroachdb/cockroach/c-deps/cryptopp'...
Cloning into '/home/tschottdorf/go/src/github.com/cockroachdb/cockroach/c-deps/cryptopp'...
mkdir -p pkg/sql/parser/gen
awk -f pkg/sql/parser/help.awk < pkg/sql/parser/sql.y > pkg/sql/parser/help_messages.go.tmp || rm pkg/sql/parser/help_messages.go.tmp
set -euo pipefail; \
TYPES=$(awk '/func.*sqlSymUnion/ {print $(NF - 1)}' pkg/sql/parser/sql.y | sed -e 's/[]\/$*.^|[]/\\&/g' | tr '\n' '|' | sed -E '$s/.$//'); \
sed -E "s_(type|token) <($TYPES)>_\1 <union> /* <\2> */_" < pkg/sql/parser/sql.y | \
awk -f pkg/sql/parser/replace_help_rules.awk | \
sed -Ee 's,//.*$,,g;s,/[*]([^*]|[*][^/])*[*]/, ,g;s/ +$//g' > pkg/sql/parser/gen/sql-gen.y.tmp || rm pkg/sql/parser/gen/sql-gen.y.tmp
awk -f pkg/sql/parser/all_keywords.awk < pkg/sql/parser/sql.y > pkg/sql/lex/keywords.go.tmp || rm pkg/sql/lex/keywords.go.tmp
awk -f pkg/sql/parser/reserved_keywords.awk < pkg/sql/parser/sql.y > pkg/sql/lex/reserved_keywords.go.tmp || rm pkg/sql/lex/reserved_keywords.go.tmp
mv -f pkg/sql/lex/keywords.go.tmp pkg/sql/lex/keywords.go
mv -f pkg/sql/parser/help_messages.go.tmp pkg/sql/parser/help_messages.go
gofmt -s -w pkg/sql/lex/keywords.go
gofmt -s -w pkg/sql/parser/help_messages.go
mv -f pkg/sql/lex/reserved_keywords.go.tmp pkg/sql/lex/reserved_keywords.go
mv -f pkg/sql/parser/gen/sql-gen.y.tmp pkg/sql/parser/gen/sql-gen.y
gofmt -s -w pkg/sql/lex/reserved_keywords.go
mv -f pkg/sql/parser/helpmap_test.go.tmp pkg/sql/parser/helpmap_test.go
gofmt -s -w pkg/sql/parser/helpmap_test.go
github.com/cockroachdb/cockroach/pkg/cmd/prereqs
yarn install v1.9.4
[1/5] Validating package.json...
[2/5] Resolving packages...
Cloning into '/home/tschottdorf/go/src/github.com/cockroachdb/cockroach/c-deps/googletest'...
[3/5] Fetching packages...
error Can't make a request in offline mode ("https://registry.yarnpkg.com/analytics-node/-/analytics-node-3.0.0.tgz")
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Makefile:268: recipe for target 'pkg/ui/yarn.installed' failed
make: *** [pkg/ui/yarn.installed] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -267,7 +267,7 @@ $(GITHOOKSDIR)/%: githooks/%
 endif
 
 .SECONDARY: pkg/ui/yarn.installed
-pkg/ui/yarn.installed: pkg/ui/package.json pkg/ui/yarn.lock pkg/ui/yarn.protobufjs-cli.lock
+pkg/ui/yarn.installed: pkg/ui/package.json pkg/ui/yarn.lock pkg/ui/yarn.protobufjs-cli.lock | bin/.submodules-initialized
 	$(NODE_RUN) -C pkg/ui yarn install --offline
 	# Prevent ProtobufJS from trying to install its own packages because a) the
 	# the feature is buggy, and b) it introduces an unnecessary dependency on NPM.

